### PR TITLE
Create contest ui 13

### PR DESCRIPTION
**Pull request by patelparth1993**
I wasn't able to login to aws with the given creds. Hence, saved images on imgur for now (thanks for that Perry). Need to add user_id from future state manager and some web responsiveness.

### CODE REVIEW

File: client/src/themes/theme.js [hunk 1]
```diff
@@ -4,12 +4,19 @@ export const theme = createMuiTheme({
   typography: {
     fontFamily: '"Poppins"',
     fontSize: 12,
+    color: "red",
```
Comment (perryraskin) [inline client/src/themes/theme.js]: Should this be here? Won't this make all text be red?

Comment (perryraskin) [reply client/src/themes/theme.js]: Also `fontFamily` has nested quotes, should probably be just `"Poppins"`?

File: client/src/components/Header.js [hunk 2]
```diff
@@ -36,28 +35,28 @@ const useStyles = makeStyles({
...
+  const [responseMessage, setResponseMessage] = useState("");
 
   const classes = useStyles();
   return (
-    <AppBar position="static" className={classes.bar}>
+    <AppBar position='static' className={classes.bar}>
       <Toolbar>
         <Grid container>
           <Grid item xs={3}>
-            <a className={classes.link} href="/">
```
Comment (iglesias294) [inline client/src/components/Header.js:49]: I'm getting a console message that you can't nest an h6 in an h6.

**Review by iglesias294 — APPROVED** [review]
Everything works and looks good. Only minor nitpick from a UX perspective is that default date/times are not set but placeholders are, so as a user if I think that the default time is already selected I get a "Please enter all required fields" message. Not important just thought it might be a good idea.

File: client/src/themes/theme.js (hunk 1 above)
Comment (patelparth1993) [reply client/src/themes/theme.js]: Added color for testing a thing but forgot to remove it. Nice catch. Fixed it.

**Review by perryraskin — APPROVED** [review]

**Pull request merged**